plugins/common/functions

```diff
@@ -814,24 +814,24 @@ get_ipv6_regex() {
   echo "${RE_IPV6}\$"
 }
 
-get_global_vhost() {
-  declare desc="return global vhost"
+get_global_vhosts() {
+  declare desc="return global vhosts"
   local GLOBAL_VHOST_FILE="$DOKKU_ROOT/VHOST"
-  [[ -f "$GLOBAL_VHOST_FILE" ]] && local GLOBAL_VHOST=$(< "$GLOBAL_VHOST_FILE")
-  echo "$GLOBAL_VHOST"
+  [[ -f "$GLOBAL_VHOST_FILE" ]] && local GLOBAL_VHOSTS=$(< "$GLOBAL_VHOST_FILE")
+  echo "$GLOBAL_VHOSTS"
 }
 
 is_global_vhost_enabled() {
   declare desc="returns true if we have a valid global vhost set; otherwise returns false"
-  local GLOBAL_VHOST=$(get_global_vhost)
-  local GLOBAL_VHOST_ENABLED=true
+  local GLOBAL_VHOSTS=$(get_global_vhosts)
+  local GLOBAL_VHOSTS_ENABLED=true
   local RE_IPV4="$(get_ipv4_regex)"
   local RE_IPV6="$(get_ipv6_regex)"
 
-  if [[ -z "$GLOBAL_VHOST" ]] || [[ "$GLOBAL_VHOST" =~ $RE_IPV4 ]] || [[ "$GLOBAL_VHOST" =~ $RE_IPV6 ]]; then
-    local GLOBAL_VHOST_ENABLED=false
+  if [[ -z "$GLOBAL_VHOSTS" ]] || [[ "$GLOBAL_VHOSTS" =~ $RE_IPV4 ]] || [[ "$GLOBAL_VHOSTS" =~ $RE_IPV6 ]]; then
```

Comment:
@josegonzalez what behavior would we expect if a user set the global host to something like
```
dokku.me
184.23.5.105
```